PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -35,7 +35,7 @@
 
 # make Fall17 the default one in Run2
 run2_egamma.toModify(photon_id_modules_WorkingPoints_nanoAOD,
-                     modules=photon_id_modules_WorkingPoints_nanoAOD_Run2.modules).\
```

Comment:
@Prasant1993 remove this extra space you have added. Looks like this is what caused the test to fail.

Reply:
@a-kapoor, thanks for spotting the additional space issue present. Fixed it now.